src/create.cpp

```diff
@@ -331,25 +369,58 @@ namespace create {
     return serial->send(cmd, 5);
   }
 
-  bool Create::driveWheels(const float& leftVel, const float& rightVel) const {
-    int16_t leftCmd = roundf(leftVel * 1000);
-    int16_t rightCmd = roundf(rightVel * 1000);
-    BOUND(leftCmd, -util::CREATE_2_MAX_VEL * 1000, util::CREATE_2_MAX_VEL * 1000);
-    BOUND(rightCmd, -util::CREATE_2_MAX_VEL * 1000, util::CREATE_2_MAX_VEL * 1000);
+  bool Create::driveWheels(const float& leftVel, const float& rightVel) {
+    float boundedLeftVel = BOUND_CONST(leftVel, -model.maxVelocity, model.maxVelocity);
+    float boundedRightVel = BOUND_CONST(rightVel, -model.maxVelocity, model.maxVelocity);
+    requestedLeftVel = boundedLeftVel;
+    requestedRightVel = boundedRightVel;
```

Comment:
We document that the requested velocities have been bounded by the maximum of the model in the [header line 461 and 465](https://github.com/AutonomyLab/libcreate/pull/25/files#diff-68cd064ce0ad0def26d0a31771014ccdR461)